utils/video.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-useless-escape */
+import fetch from 'isomorphic-unfetch'
+
+export const isYouTubeVideo = videoUrl =>
+  videoUrl.includes('youtube') || videoUrl.includes('youtu.be')
+
+export function getYoutubeVideoId(videoUrl) {
+  const youtubeVideoIdRegex = /^.*(youtu.be\/|v\/|e\/|u\/\w+\/|embed\/|v=)([^#\&\?]*).*/
+  const match = youtubeVideoIdRegex.exec(videoUrl)
+  const videoId = !!match && match.length > 2 ? match[2] : ''
+
+  return videoId
+}
+
+export function getVimeoId(videoUrl) {
+  const vimeoVideoIdRegex = /(http|https)?:\/\/(www\.|player\.)?vimeo.com\/(?:video\/(?:\w+\/)?|channels\/(?:\w+\/)?|groups\/([^\/]*)\/videos\/|)(\d+)/
+  const match = vimeoVideoIdRegex.exec(videoUrl)
+  const videoId = !!match && match.length > 4 ? match[4] : ''
+
+  return videoId
+}
+
+export function getYoutubeThumbnail(videoUrl) {
+  const videoId = getYoutubeVideoId(videoUrl)
+
+  // http://img.youtube.com/vi/[video-id]/[thumbnail-number].jpg
+  return `http://img.youtube.com/vi/${videoId}/0.jpg`
+}
+
+// It is only covering the happy path
+// Unlikely that vimeo will be down
```

Comment:
Could we at least return a placeholder image if the request fails?

Reply:
Can be ignored since the site doesn't break.